index.html

```diff
@@ -824,10 +824,11 @@ <h1><span style="font-size:1.0em">Example</span></h1>
               <ul>
                 <li style="text-align:left"><a href="http://presentation.cloudalchemy.org" target="_blank">presentation.cloudalchemy.org</a></li>
                 <li style="text-align:left"><a href="https://github.com/cloudalchemy" target="_blank">github.com/cloudalchemy</a></li>
-                <li style="text-align:left"><a href="https://prometheus.io/docs" target="_blank">prometheus.io/docs</a></li>
-                <li style="text-align:left"><a href="https://www.safaribooksonline.com/library/view/prometheus-up/9781492034131" target="_blank">www.safaribooksonline.com/library/view/prometheus-up/9781492034131</a></li>
-                <li style="text-align:left"><a href="https://prometheus.io/webtools/alerting/routing-tree-editor/" target="_blank">prometheus.io/webtools/alerting/routing-tree-editor</a></li>
-                <li style="text-align:left"><a href="https://prometheusbook.com/" target="_blank">prometheusbook.com</a></li>
+                <li style="text-align:left"><a href="https://github.com/juliusv/ansible_prometheus_demo" target="_blank">A demo setup of Prometheus using CloudAlchemy's Ansible roles</a></li>
```

Comment:
"A demo setup of Prometheus in HA mode using..."

Basic demo of prometheus using cloudalchemy roles is cloudalchemy/demo-site :)